python/test/unit/language/test_core.py

```diff
@@ -4535,6 +4527,8 @@ def sort_kernel(X, Z, N: tl.constexpr, M: tl.constexpr, descending: tl.constexpr
 @pytest.mark.parametrize("M, N", [[1, 512], [8, 64], [256, 16], [512, 8]])
 @pytest.mark.parametrize("dtype_str", ['int32', 'float16', 'float32'])
 def test_flip(M, N, dtype_str, device):
+    if is_hip():
+        pytest.skip('test_flip for HIP currently not working')
```

Comment:
it surprising that this fails as it doesn't require any new backend features. 

General question: Is this a filtering you could do in your CI command line instead? By that I mean that changing the test file for every test you want disabled because things are work in progress is probably not ideal and you can probably filter it out in CI doing something like -k 'not test_flip'?

Reply:
yeah, I can do that

Reply:
Thanks!

Reply:
ok skipped test_flip using pytest. I will use that to skip new failures moving forward. Just to limit the scope of this PR, I will leave the other old is_hip() skips in test_core for now but we will remove it in the subsequent prs soon